docs/api_definitions.md

```diff
@@ -98,9 +98,9 @@ These are features which are only available to tyk PRO users
 | Security - Allow list | ⚠️ | [Untested #92](https://github.com/TykTechnologies/tyk-operator/issues/93) |
 | Security - Block list | ⚠️ | [Untested #92](https://github.com/TykTechnologies/tyk-operator/issues/93) |
 | Security - Ignore list | ⚠️ | [Untested #92](https://github.com/TykTechnologies/tyk-operator/issues/93) |
-| Transform - Internal | ⚠️ | [Untested #93](https://github.com/TykTechnologies/tyk-operator/issues/93) |
-| Transform - Method | ⚠️ | [Untested #93](https://github.com/TykTechnologies/tyk-operator/issues/93) |
-| Transform - Mock | ⚠️ | [Untested #93](https://github.com/TykTechnologies/tyk-operator/issues/93) |
+| Transform - Internal | ⚠️ | Untested |
+| Transform - Method |  ✅ | - |
```

Comment:
Just one thing, can we provide a link to the example?

```
[Transform - Method](../bdd/custom_resources/transform/method.yaml)
```

Reply:
done 6b4f587